swim/labels.go

```diff
@@ -1,5 +1,48 @@
 package swim
 
+import (
+	"errors"
+	"strings"
+)
+
+var (
+	// labelsBannedKeyCharacters keeps a list of all banned occurances for label keys
+	labelsBannedKeyCharacters = []string{
+		// the equal sign is used in label checksum calculation. To prevent
+		// unintended or forced collisions on the checksums of labels we do not
+		// allow the equal sign to be used in the key portion of a label
+		"=",
```

Comment:
Well, we don't know how customers are going to use our labels. So I would rather be very prescriptive (e.g. allow only `[a-zA-Z0-9_-]`), or be not prescriptive at all - allow everything.

It feels backward to disable only one symbol just because we don't feel customers will be using them. What if they want their keys for some reason to be base64-encoded movie titles?